admin/services/check-start.service/check-start.service.js

```diff
@@ -194,6 +201,7 @@ checkStartService.initialisePupilCheck = async function (
   availableForms,
   usedFormIds,
   isLiveCheck,
+  userId,
   schoolId = null,
```

Comment:
Not relevant to my PR but this seems to be wrong. `schoolId` will definitely not be null at that stage (the caller method already evaluates that) so providing a default to null seems to be totally wrong here.

Reply:
please check existing call sites and remove default if not required